components/post_view/reaction/reaction.test.tsx

```diff
@@ -4,18 +4,23 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
+import {Reaction as ReactionType} from 'mattermost-redux/types/reactions';
+
 import Reaction from 'components/post_view/reaction/reaction';
+import {TestHelper} from 'utils/test_helper';
 
 describe('components/post_view/Reaction', () => {
-    const post = {id: 'post_id_1'};
-    const reactions = [{user_id: 'user_id_2'}, {user_id: 'user_id_3'}];
+    const post = TestHelper.getPostMock({
+        id: 'post_id',
+    });
+    const reactions: ReactionType[] = [{user_id: 'user_id_2', post_id: post.id, emoji_name: ':smile:', create_at: 0}, {user_id: 'user_id_3', post_id: post.id, emoji_name: ':smile:', create_at: 0}];
     const emojiName = 'smile';
     const actions = {
         addReaction: jest.fn(),
         getMissingProfilesByIds: jest.fn(),
         removeReaction: jest.fn(),
     };
-    const currentUserId = 'user_id_1';
+    const currentUserId = TestHelper.getUserMock({id: 'user_id_1'}).id;
```

Comment:
Why do this?

Reply:
I was also working to migrate another test and wanted to be consistent in terms of creating mock user. But you're right, since the prop is expecting a string, there is no need to complicate the things. I reverted the changes